system/jlib/jsmartsock.cpp

```diff
@@ -45,9 +45,22 @@ class SmartSocketListParser
         unsigned port = defport;
 
         char *saveptr;
-        char *ip = strtok_r(copyFullText, "|", &saveptr);
-        while (ip != NULL)
+        char *ip0 = strtok_r(copyFullText, "|", &saveptr);
+        while (ip0 != NULL)
         {
+            // strip off http[s]://
+            char *ip = strstr(ip0, "http://");
```

Comment:
I can't remember the call, but I think there's a jlib function to split a connection string into prot;user;pass;host;port;uri, might be worth exploring

Reply:
I don't know of a routine to handle the ip range like this does (ie 1.2.3.4-6:8000).